src/lib/__test__/numbers.spec.ts

```diff
@@ -99,7 +99,7 @@ describe('getFractionDigits', () => {
   });
   it('trims trailing 0s', () => {
     // eslint-disable-next-line prettier/prettier
-    expect(getFractionDigits(0.00500)).toEqual(3);
+    expect(getFractionDigits(0.005)).toEqual(3);
```

Comment:
oh nooo

Reply:
the linter did this, not meeee

Reply:
I run the linter on all files pretty often and my vscode does on save, I don't know how to make it keep these extra 0s

Reply:
`command shift p > save without formatting` will let you save this file without formatting

unsure why your linter is ignoring the eslint disable 😠

Reply:
it's not the linter that fixes - it's prettier

Reply:
I run prettier on all ts files before most prs